Library/Homebrew/dev-cmd/update-license-data.rb

```diff
@@ -28,13 +28,13 @@ def update_license_data
 
     SPDX.download_latest_license_data!
 
-    Homebrew.failed = system("git", "diff", "--stat", "--exit-code", SPDX::JSON_PATH) if args.fail_if_not_changed?
+    Homebrew.failed = system("git", "diff", "--stat", "--exit-code", HOMEBREW_DATA_PATH) if args.fail_if_not_changed?
 
     return unless args.commit?
 
     ohai "git add"
-    safe_system "git", "add", SPDX::JSON_PATH
+    safe_system "git", "add", HOMEBREW_DATA_PATH
```

Comment:
Not blocking, but this might cause issues in the future if we add new files to the `data` directory

Reply:
Is it worth moving license data to `HOMEBREW_DATA_PATH/"spdx/"`?

`brew update-license-data` is probably only called with `--fail-if-not-changed` or `--commit` by CI which should always start from a clean state, right? The only changes should be the license update.

Reply:
I agree that `git add` should add only that specific file for directory specific to SPDX.

Reply:
> `brew update-license-data` is probably only called with `--fail-if-not-changed` or `--commit` by CI which should always start from a clean state, right? The only changes should be the license update.

Yeah I don't think this will cause problems for the time being, but who knows, it might in a few months or years from now

> Is it worth moving license data to `HOMEBREW_DATA_PATH/"spdx/"`?

Maybe, see Rylan12/brew#1